content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -136,6 +137,31 @@ done
 
 Once you successfully upgrade to version v1.6.0, you will be able to view information about engine image versions on the UI.
 
+### Danger Zone Setting Configuration
+
+Starting with Longhorn v1.6.0, Longhorn allows you to promptly update the [Danger Zone settings](https://longhorn.io/docs/1.6.0/references/settings/#danger-zone) without the need to wait for all volumes detached. This enhancement is designed to provide users with more flexibility and efficiency in managing their configurations. To achieve the immediate effect, one of the following conditions must be met:
+
+- No Attached Volumes: If there are no volumes attached, the setting change will take effect immediately.
+- Engine Image Upgrade (Volume Live Upgrade): During a volume live upgrade, which involves creating a new instance manager pod, the setting change will take effect immediately for the new pod.
+
+If immediate effect conditions aren't met, the setting change is reconciled hourly, taking effect when all volumes are detached.
+
+- When all volumes are detached, the settings in the following table are immediately applied and the system-managed components including engine-image, instance-manager, longhorn-csi-plugin, csi-attacher, csi-provisioner, csi-resizer, and csi-snapshotter. are restarted. If you do not detach all volumes before the settings are synchronized, the settings are not applied immediately as mentioned above and you might need to reconfigure the settings after detaching the remaining volumes; otherwise, you can wait for the setting change to be reconciled in an hour.
```

Comment:
```suggestion
When all volumes are detached, the settings in the following table are immediately applied and the system-managed components including engine-image, instance-manager, longhorn-csi-plugin, csi-attacher, csi-provisioner, csi-resizer, and csi-snapshotter. are restarted. If you do not detach all volumes before the settings are synchronized, the settings are not applied immediately as mentioned above and you might need to reconfigure the settings after detaching the remaining volumes; otherwise, you can wait for the setting change to be reconciled in an hour.
```

Reply:
Updated, removed the `-`. Thanks